### DISCUSSION TITLE
Support for Water-Temperature sensors...

### DISCUSSION THREAD

**Discussion by EyoSama** [devcompl/TTController#183] [Configuration] [answered]
Hello Moshi,

I was wondering if it is possible to support sensors such as the Aquacomputer Highflow NEXT, which measures water temp, flow, conductivity, and quality. It would of course be great if we could control the fan speeds based on the water temperature of the loop, rather than basing it on the CPU and GPU usage. In the end, it's just the water temps that actually matter in a water-cooled system, since that controls heat removal from the GPU and CPU.

I also couldn't find any of the Water_IN temp sensors in Report in your service. The Water_IN sensor is present on most high-end motherboards. Is this something that can be enabled via your software?

Thanks!

Eyo...

**Comment by MoshiMoshi0** [comment, marked as answer]
The sensor report shows all available sensors to use. 
Sensor data is provided by [LibreHardwareMonitor/LibreHardwareMonitor](https://github.com/LibreHardwareMonitor/LibreHardwareMonitor) so you would have to make your feature requests there.

**Reply by EyoSama** [reply]
Thanks, I'll check there.

What I'm not seeing in the sensors that are read, is any information on WATER_IN or WATER_OUT, which are part of the ASUS (and other) motherboards. Would this also be something to check with them, or is that something you can control on your side?

**Reply by MoshiMoshi0** [reply]
TTController can only use what is visible in LibreHardwareMonitor, so you can download latest LibreHardwareMonitor release and see what sensors are visible in their app. 
If its not visible there then you have to make a request on their repo.

**Reply by EyoSama** [reply]
Cool, thanks for the response. I went ahead and made a request on the LHM page. Cheers!
